doc/source/whatsnew/v0.17.0.txt

```diff
@@ -31,6 +31,45 @@ New features
 Other enhancements
 ^^^^^^^^^^^^^^^^^^
 
+- ``DatetimeIndex`` can be instantiated using strings contains ``NaT`` (:issue:`7599`)
+- ``Timestamp`` and ``to_datetime`` can parse the same formats as ``DatetimeIndex``. (:issue:`7599`)
```

Comment:
Maybe make this: "The string parsing of to_datetime, Timestamp and DatetimeIndex has been made consistent"? (seems a clearer message to the users to me)